### ISSUE TITLE
make html fails

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#485]
Hi, I have run sphinx successfully on my local computer. Then uploaded my docs to a remote server, and tried to run make html there, but get the following error on the remote machine.

```
#!python

# Sphinx version: 1.0.1
# Docutils version: 0.5 release
# Jinja2 version: 2.5
Traceback (most recent call last):
  File "/home/akasha/lib/python2.5/sphinx/cmdline.py", line 172, in main
    warningiserror, tags)
  File "/home/akasha/lib/python2.5/sphinx/application.py", line 136, in __init__
    self._init_builder(buildername)
  File "/home/akasha/lib/python2.5/sphinx/application.py", line 196, in _init_builder
    __import__('sphinx.builders.' + mod, None, None, [cls]), cls)
  File "/home/akasha/lib/python2.5/sphinx/builders/html.py", line 33, in <module>
    from sphinx.util import jsonimpl, copy_static_entry
  File "/home/akasha/lib/python2.5/sphinx/util/jsonimpl.py", line 42, in <module>
    load = json.load
AttributeError: 'NoneType' object has no attribute 'load'
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/485
- Originally reported by: Viktor Nagy
- Originally created at: 2010-08-04T10:50:25.399

**Comment by shimizukawa** [comment]
_From Viktor Nagy on 2010-08-04 10:22:24+00:00_

The error was solved by installing simplejson. It's weird though that some requirements were not installed by pip.
I'm running this on a webfaction server where pip under python2.5 and installs stuff under my home directory.

**Comment by shimizukawa** [comment]
_From Santoso Wijaya on 2010-08-04 20:30:02+00:00_

This is a duplicate of <<issue 476>>. simplejson should not be required for `make html` to work. Fixed by <<cset a1fd9830e57944b980629ad3bf5b11771dc0af32>>.

Duplicate of #476.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-10-18 07:29:32+00:00_

Removing version: 1.0.1 (automated comment)
